app/javascript/dashboard/components-next/message/bubbles/Email/Index.vue

```diff
@@ -135,6 +197,14 @@ const hasQuotedMessage = computed(() => {
         </button>
       </div>
     </section>
+    <span v-if="hasTranslations" class="py-2 px-3">
+      <span
+        class="text-xs text-n-slate-11 cursor-pointer hover:underline"
+        @click="handleSeeOriginal"
+      >
+        {{ viewToggleText }}
+      </span>
+    </span>
```

Comment:
Make this a small component which accepts the props show, whether it is showing the original and provide action which responds to clicks